solutions/gold/ys-QueueComposite.mdx

```diff
@@ -0,0 +1,93 @@
+---
+id: ys-QueueComposite
+source: YS
+title: Queue Composite
+author: Ryan Chou
+---
+
+
+## Explanation
+
+Note that we can compute the portion with $x$ as a factor independently from the constant in the equation. Thus, we can support $\mathcal{O}(\log n)$ removals, and $\mathcal{O}(1)$ additions and queries.
```

Comment:
is this...really all there is?

Reply:
that was my thought process 😳 

is there a different intended solution? i couldn't find any other solutions online...

Reply:
seems like one of those one-line editorials where the reader ends up having more questions than answers

Reply:
word